external/fv3fit/tests/test_stacking.py

```diff
@@ -137,6 +137,16 @@ def test__shuffled_dask():
     shuffled(np.random.RandomState(1), [dataset])
 
 
+@pytest.mark.parametrize(("a_chunk_size", "b_chunk_size"), [(10, 5), (5, 10)])
+def test__shuffled_dask_different_chunks(a_chunk_size, b_chunk_size):
```

Comment:
I can't tell for sure what this is testing, is it that `shuffled` maintains the chunking of a dataset with multiple chunk sizes? Can you make the name more verbose or add a docstring?